elasticsearch/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash 
+
+set -ex
+
+rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch
+yum install -y --setopt=tsflags=nodocs \
+  java-1.8.0-openjdk \
+  elasticsearch
+yum clean all
+
+# need these so elasticsearch doesn't reject its config while installing plugins
+export SERVICE_DNS=placeholder
+export CLUSTER_NAME=placeholder
+
+mkdir -p ${HOME}
+ln -s /usr/share/elasticsearch /usr/share/java/elasticsearch
+/usr/share/elasticsearch/bin/plugin -i com.floragunn/search-guard/0.5
+#/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/jcantril/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
+/usr/share/elasticsearch/bin/plugin -i openshift-elasticsearch-plugin -u http://file.rdu.redhat.com/lmeyer/openshift-elasticsearch-plugin-0.4-SNAPSHOT.zip
```

Comment:
should probably change this to be the proper location, we can update the version after we publish our latest